codeListAPI/get_edition_test.go

```diff
@@ -0,0 +1,59 @@
+package codeListAPI
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/gavv/httpexpect"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestSuccessfulRetrievalOfEdition(t *testing.T) {
+
+	if err := ds.DropCodelists(); err != nil {
+		t.Error(err)
+		t.Fail()
+	}
+
+	if err := ds.SetupCodelistEditions(); err != nil {
+		t.Error(err)
+		t.Fail()
+	}
+
+	codelistAPI := httpexpect.New(t, cfg.CodeListAPIURL)
+
+	Convey("Given an edition of a codelist eqists", t, func() {
+		Convey("When a request is made for the edition", func() {
+			Convey("Then the codelist is returned to the user", func() {
+				response := codelistAPI.GET("/code-lists/ABCDEF/editions/2018").Expect().Status(http.StatusOK).JSON().Object()
+
+				response.Value("id").Equal("ABCDEF")
```

Comment:
I'm not certain we should be returning ID as the code list name here. not sure ID is a meaningful field for this endpoint.